nautobot/extras/tests/integration/test_plugins.py

```diff
@@ -28,9 +28,9 @@ class PluginWebhookTest(SeleniumTestCase):
 
     def setUp(self):
         super().setUp()
-        for f in os.listdir("/tmp"):
+        for f in os.listdir("/tmp"):  # noqa: S108
             if f.startswith("test_plugin_webhook_"):
-                os.remove(os.path.join("/tmp", f))
+                os.remove(os.path.join("/tmp", f))  # noqa: S108
```

Comment:
This is probably wrong, we should probably be using `tempfile` APIs here instead of hard-coding `/tmp`.